packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1933,23 +1933,6 @@ abstract class BuildContext {
   /// ancestor is added or removed).
   InheritedWidget inheritFromWidgetOfExactType(Type targetType);
 
-  /// Obtains the element corresponding to the nearest widget of the given type,
-  /// which must be the type of a concrete [InheritedWidget] subclass.
-  ///
-  /// Calling this method is O(1) with a small constant factor.
-  ///
-  /// This method does not establish a relationship with the target in the way
-  /// that [inheritFromWidgetOfExactType] does. It is normally used by such
-  /// widgets to obtain their corresponding [InheritedElement] object so that they
-  /// can call [InheritedElement.dispatchDidChangeDependencies] to actually
-  /// notify the widgets that _did_ register such a relationship.
-  ///
-  /// This method should not be called from [State.deactivate] or [State.dispose]
-  /// because the element tree is no longer stable at that time. To refer to
-  /// an ancestor from one of those methods, save a reference to the ancestor
-  /// by calling [inheritFromWidgetOfExactType] in [State.didChangeDependencies].
-  InheritedElement ancestorInheritedElementForWidgetOfExactType(Type targetType);
```

Comment:
we should keep this one, because it has important use cases (for example, getting the locale from an event handler that sends an accessibility event -- the build method doesn't need to be called if the locale changed, since it doesn't depend on it, and there's nothing to update if the locale changes.)

Reply:
Returned.